### PR TITLE
[WIP] front end of comments

### PR DESCRIPTION
**Pull request by MandyMeindersma**

### CODE REVIEW

File: react/src/js/react/posts/Post.jsx [hunk 1]
```diff
@@ -69,9 +69,43 @@ class Post extends Reflux.Component {
         return contentList;
     }
 
-    // TODO
+    // TODO add actual author and coment here
     renderComments() {
-        return null;
+        console.log(this.currentPost.comments);
```
Comment (amalik2) [inline react/src/js/react/posts/Post.jsx]: This should be done in PostView.jsx instead

File: react/src/js/react/posts/Post.jsx [hunk 2]
```diff
@@ -69,9 +69,43 @@ class Post extends Reflux.Component {
...
+        const comments = this.currentPost.comments.map((comment) => (
+            <p key={comment}>
+                {comment}
+            </p>
+        ));
+        return comments;
+    }
+
+    handleCommentSubmit = (e) => {
```
Comment (amalik2) [inline react/src/js/react/posts/Post.jsx]: This should be done in PostView.jsx instead

File: react/src/js/react/discover/PostsStore.js [hunk 3]
```diff
@@ -21,6 +22,9 @@ export class PostsStore extends Reflux.Store {
             creatingPost: false,
             successfullyCreatedPost: false,
             failedToCreatePost: false,
+            updatingPost: false,
```
Comment (amalik2) [inline react/src/js/react/discover/PostsStore.js]: For clarity, this should be "creatingComment"
The other state attributes you added should have Comment in their name, as opposed to Post

File: react/src/js/react/discover/PostsStore.js [hunk 4]
```diff
@@ -7,7 +7,8 @@ export const PostsActions = Reflux.createActions([
     "createPost",
     "getPosts",
     "deletePost",
-    "getPost"
+    "getPost",
+    "postComment"
```
Comment (chaitali) [inline react/src/js/react/discover/PostsStore.js]: You should also change this "addComment" instead

**Comment by MandyMeindersma** [comment]
lol I fixed all the things and then I tried to close and open this PR to kick off the build again and now I am not allowed to reopen it (probably because I force pushed to the branch, idk, new PR here though: #76

**Pull request closed without merging**